crates/matrix-sdk/src/client/builder.rs

```diff
@@ -519,6 +519,15 @@ impl ClientBuilder {
     }
 }
 
+/// Creates a server name from a user supplied string. The string is first
+/// sanitized by removing whitespace, the http(s) scheme and any trailing
+/// slashes before being parsed.
+pub(crate) fn sanitize_server_name(s: &str) -> crate::Result<OwnedServerName, IdParseError> {
```

Comment:
Where is this being called from?

Reply:
Good catch, it's just locally in the build function. Can remove that pub(crate) now 👍